src/app/config/confparse.c

```diff
@@ -534,11 +534,17 @@ config_var_is_contained(const config_var_t *var)
 {
   return struct_var_is_contained(&var->member);
 }
+
+/**
+ * Return true iff variable <b>var</b> should appear on list of variables
```

Comment:
It might be clearer if we say:
"iff variable <b>var</b> should appear on list of variable names"

Because is_gettable() is about fetching values.

Reply:
6d6c70ba4e4f2d19e9df3ba130e5fb2ef3d9acb9 is the fix commit here -- it's not a fixup, because of rebase conflicts.

Reply:
.